shared/src/license.ts

```diff
@@ -5,6 +5,7 @@ export type LicenseStatusItem = {
 
 export type LicenseStatusResult = {
   status: LicenseStatus;
+  expirationISO?: string;
```

Comment:
Why should this be potentially undefined? When would it ever not exist but the items and status still do?